model/src/pyrenew/deterministic/deterministic.py

```diff
@@ -28,13 +30,18 @@ def __init__(
             A tuple with arraylike objects.
```

Comment:
```suggestion
            An arraylike object.
```

Reply:
We should discuss elsewhere whether `DeterministicVariable`s should be able to have an arbitrary `pytree` as their `vars` value. I lean yes.

I would also support renaming `vars` to `value`, for consistency with `SampledValue`.

For clarity, both of the above are out of scope for this PR.

Reply:
Issue created